blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -91,7 +91,8 @@ def big_sql(url):
     except sa.exc.OperationalError as e:
         pytest.skip(str(e))
     else:
-        t = odo(zip(list('a'*100), list(range(100))), t)
+        t = odo(zip(list("".join(['a'*25, 'b'*25, 'c'*25, 'd'*25])),
```

Comment:
style nit: can we put spaces around binary operators, without them expressions are very crowded